### PR TITLE
release v1.11.0

### PR DESCRIPTION
**Pull request by zachferland**

### CODE REVIEW

**Comment by zachferland** [comment]
To be released tmrrw

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Release Notes
 
+## v1.11.0 - 2019-01-15
+Includes performance improvements that will allow syncing to start sooner on client
+and for onSync done to resolve more quickly.
+
+* feat: feat: cache has entries and send on message
```
Comment (oed) [inline RELEASE-NOTES.md]: typo, "feat:" twice :)

Comment (oed) [reply RELEASE-NOTES.md]: Other than that, this should be good for release :)

Comment (zachferland) [reply RELEASE-NOTES.md]: will fix that 👍

**Comment by zachferland** [comment]
To be merged after cache resources is deployed in production

**Comment by zachferland** [comment]
Going to do some quick runs in staging, then terraform, then release

**Comment by zachferland** [comment]
Found bug with 0 being falsy, push fix in pr soon

**Comment by zachferland** [comment]
will be rebased on https://github.com/3box/3box-pinning-node/pull/274 before release

**Pull request merged**